org.eclipse.jdt.core.manipulation/proposals/org/eclipse/jdt/internal/ui/text/correction/proposals/RefactoringCorrectionProposalCore.java

```diff
@@ -67,10 +67,10 @@ public TextChange createTextChange() throws CoreException {
 		Change o = fRefactoring.createChange(new NullProgressMonitor());
 		if(o instanceof TextChange)
 			return (TextChange) o;
-		if( o instanceof CompositeChange) {
+		if (o instanceof CompositeChange) {
```

Comment:
Whitespace changes without anything else and it even changes one line to certain formatting but leaves the previous one with different style. Please drop the whole file from this PR as there is nothing but this.